quipucords/scanner/network/connect.py

```diff
@@ -128,8 +129,21 @@ def __init__(self, scan_job, scan_task):
         super().__init__(scan_job, scan_task)
         self.scan_task = scan_task
 
-    def run(self):
+    def run(self, manager_interrupt):
         """Scan network range ang attempt connections."""
```

Comment:
I know you didn't write the docstring but if you feel like it, change ang --> and

Reply:
ang was in there 3 other places